### ISSUE TITLE
The Clarity Icon under Custom Triggers does not receive keyboard focus

### ISSUE THREAD

**Issue by chlane** [vmware/clarity#4616]
[Clarity Version]
3.1.1

[Test URL]
https://clarity.design/documentation/signposts#examples

[WCAG 2.1 Guideline]
2.1.1 Keyboard 
(Level A)
https://www.w3.org/TR/WCAG21/#keyboard

[OS]
MacOS 10.15.3, Windows 10

[User Agents/Assistive Technologies]
Safari 13.0.5, Firefox 75.0, Switch

[Component]
Signposts

[Description]
The Clarity Icon under Custom Triggers does not receive keyboard focus

[User Impact]
Switch users cannot navigate to the Clarity Icon via keyboard and must simulate a mouse click. 

[Steps to Reproduce]
See the following instructions for switch usage then follow the steps below.

Windows (see Scan through keys)
https://support.microsoft.com/en-us/help/10762/windows-use-on-screen-keyboard

Mac
https://support.apple.com/en-us/HT202865

1. Using a switch, try navigating to the Clarity Icon with the Tab key. 
For MacOS only: 
2. Navigate to and activate Pointer --&gt; Mouse Click. 
3. Select the vertical area of the click, and then the vertical coordinate of the icon.
4. Select the horizontal area of the click, and then the horizontal coordinate of the icon. 

[Recommendation]
The CLR button does not have a tabindex attribute, and is not a natively interactive element. Adding a tabindex attribute with a value of &quot;0&quot; and updating the role attribute to be &quot;button&quot; will both ensure this element receives focus, but also that its role is properly announced to users.	&lt;clr-icon _ngcontent-clarity-c224=&quot;&quot; shape=&quot;avatar&quot; clrsignposttrigger=&quot;&quot; class=&quot;signpost-trigger is-solid has-badge-info&quot; role=&quot;button&quot; aria-label=&quot;Signpost Toggle&quot; aria-expanded=&quot;false&quot; aria-controls=&quot;clr-id-1&quot; tabindex=&quot;0&quot;&gt;...&lt;/clr-icon&gt;

**Comment by envyN** [comment]
also needs to open and close the popup.

**Comment by gnomeontherun** [comment]
Hi there, this is an automated message.

In an effort to dedupe and track accessibility reports consistently, this issue is being closed and tracked internally with our Accessibility team. It will still be triaged and addressed as part of ongoing work.
